lib/dal/src/attribute/value.rs

```diff
@@ -2252,34 +2256,77 @@ impl AttributeValue {
         Ok(())
     }
 
-    pub async fn subscribe(
+    pub async fn set_to_subscriptions(
```

Comment:
It would be good to have a doc comment here emphasizing that it will overwrite any existing `AttributePrototype` and `AttributePrototypeArguments`, so any function choice, and any existing arguments will be lost. So, if the function was customized, or arguments were added that weren't subscriptions there is work to do before and after calling this function to preserve any of those if desired. And that there's still work to do before calling if the intent is to append new subscriptions, instead of wholesale replacement (like the work that the SDF endpoint does if `keep_existing_subscriptions` is `true`).

Reply:
:+1: